Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+
+const buttonNums = ['7', '8', '9', '4', '5', '6', '1', '2', '3'];
+const operators = ['÷', 'x', '-', '+', '='];
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="calculator">
+      <CalculatorDisplay text="0"/>
+      <div className="calculator-buttons">
+        <div className="number-container">
+          <ActionButton buttonStyle="action-btn" text="clear"/>
+          {buttonNums.map(number => {
+              return <NumberButton buttonStyle="number-btn" text={number}/>;
+            }
+          )}
+          <ActionButton buttonStyle="action-btn" text="0"/>
+        </div>
+        <div className="operator-buttons">
+          {operators.map(operator => {
+            return <NumberButton buttonStyle="operator-btn" text={operator}/>;
```

Comment:
To get rid of console log warning about `Each child in a list should have a unique "key" prop.`, give the the NumberButton component a `key` props